bubble-wrap-style.yaml

```diff
@@ -2644,134 +2481,376 @@ layers:
                             #priority: 48
                             text:
                                 offset: [0px, 1px]
-                                font:
-                                    fill: [0.308,0.308,0.308]
-                                    size: [[7,7px],[13,9px],[15,10px]]
-                    motorway:
```

Comment:
I remember California being special (like France) where some of the state roads needed different shield visibility than default. Please show before/after screenshots.